pkg/kubectl/cmd/get.go

```diff
@@ -513,3 +537,50 @@ func RunGet(f cmdutil.Factory, out, errOut io.Writer, cmd *cobra.Command, args [
 	cmdutil.PrintFilterCount(errOut, len(objs), filteredResourceCount, len(allErrs), filterOpts, options.IgnoreNotFound)
 	return utilerrors.NewAggregate(allErrs)
 }
+
+func cmdSpecifiesOutputFmt(cmd *cobra.Command) bool {
+	return cmd.Flags().Lookup("output") != nil && cmdutil.GetFlagString(cmd, "output") != ""
```

Comment:
cmdutil.GetFlagString(...) does a FATAL if the given flag is not defined. "output" flag is defined for the Get command, so its a bit more defensive check in this case. So can be removed, but doesn't hurt also.